Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -0,0 +1,139 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public final class EditorialProjectsViewController: UIViewController {
+  // MARK: - Properties
+
+  private lazy var closeButton: UIButton = {
+    UIButton(type: .custom)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var discoveryPageViewController: DiscoveryPageViewController = {
+    DiscoveryPageViewController.configuredWith(sort: .newest)
+      |> \.preferredBackgroundColor .~ .clear
+  }()
+
+  private lazy var headerView: UIView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let viewModel: EditorialProjectsViewModelType = EditorialProjectsViewModel()
+
+  // MARK: - Lifecycle
+
+  public override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.configureSubviews()
+    self.setupConstraints()
+
+    self.closeButton.addTarget(
+      self,
+      action: #selector(EditorialProjectsViewController.closeButtonTapped),
+      for: .touchUpInside
+    )
+
+    // Will be moved to be a testable output of VC that presents this one
+    self.viewModel.inputs.configure(with: .goRewardless)
```

Comment:
As the comment says - once we have the navigation flow for this we would move this configuration and write a test for that output.